### ISSUE TITLE
Remove double calculation of half codesy fee and replace with single codesy fee

### ISSUE THREAD

**Issue by kendallcorner** [codesy/codesy#615]
The codesy fee is calculated by half twice. Replacing with only once will reduce rounding error.

**Comment by jdungan** [comment]
@kendallcorner Is this included in your PR?

**Comment by kendallcorner** [comment]
No. I looked at it and it looked too complicated to wrap into that PR, so I left it as 2 half fees like you had before.

**Comment by kendallcorner** [comment]
Also, I was going to look at it later, that's why the issue, but not sure when exactly 'later' is.
